src/packages/core/src/server.ts

```diff
@@ -283,6 +292,19 @@ export class Server<
     }
   }
 
+  public address() {
+    if (this.#listenSocket) {
```

Comment:
My first iteration of this checked the server's `#status` to see if we were "Open". This `this.#listenSocket` check is effectively the same (the two properties are always set together). I chose this check instead only because passing `null` to `us_socket_local_port` triggers what seems to be undefined behavior in the uWS library, so this is slightly more explicit and less prone to bugs due to future refactoring.